setup.cfg

```diff
@@ -1,2 +1,2 @@
 [metadata]
-description-file = README.md
\ No newline at end of file
+description_file = README.md
```

Comment:
guess this should be: 
`long_description = file:`

Reply:
yes changed